### PR TITLE
New navigation title dropdown control

### LINKED ISSUE #1264: Drop down inbox title to select per-repo notifications
Got added to some FB repos and now understand the pain about filtering. Was thinking of a control where you tap the title and am overlay+dropdown appear with a list of repos. Select a repo and push a new inbox VC that only loads notifications for that repo.

Thoughts?

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.16.0 (1513090523)
Device: iPhone X (iOS 11.2)
TestFlight: true
</pre>
</details>

### PR DESCRIPTION
**Pull request by rnystrom**
- Same title+subtitle as today
- New chevron indicating that its a control
- Show `UIAlertAction`s on tap
- Replace issue/PR "..." with share icon and just present share
- Update Bookmark assets to be same size as share icon

Fixes #1264

Prereq for #1267

### CODE REVIEW

**Comment by rnystrom** [comment]
simulator screen shot - iphone x - 2018-01-13 at 15 52 13

simulator screen shot - iphone x - 2018-01-13 at 15 52 15

**Comment by BasThomas** [comment]
Lovely!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Classes/Views/NavigationTitleDropdownView.swift [hunk 1]
```diff
@@ -0,0 +1,121 @@
+//
```
Comment (BasThomas) [inline Classes/Views/NavigationTitleDropdownView.swift]: Is this guy accessible at this moment? I don't think it's being recognised as a `Button` now.

Comment (rnystrom) [reply Classes/Views/NavigationTitleDropdownView.swift]: Crap! Nope, it needs some AX love for sure.

Comment (BasThomas) [reply Classes/Views/NavigationTitleDropdownView.swift]: Want to take it on now or should I create a ticket?

Comment (rnystrom) [reply Classes/Views/NavigationTitleDropdownView.swift]: @BasThomas mind filing an issue?

**Pull request merged**